wsmaster/codenvy-hosted-api-audit/src/main/java/com/codenvy/api/audit/server/AuditManager.java

```diff
@@ -0,0 +1,196 @@
+
+        //TODO: use UserManager when jpa-integration will be merged into master.
+        this.adminUserDao = adminUserDao;
+        this.workspaceManager = workspaceManager;
+        this.permissionManager = permissionManager;
+        this.licenseManager = licenseManager;
+    }
+
+    /**
+     * Prints audit report to given file.
+     *
+     * @throws ServerException
+     *         if an error occurs
+     */
+    public File generateAuditReport() throws ServerException {
+        String dateTime = new SimpleDateFormat("dd/MM/yyyy-HH:mm:ss").format(new Date());
+        final File report = new File(Files.createTempDir(), "report_" + dateTime + ".txt");
+
+        try {
+            printHeader(report);
+            printAllUsersInfo(report);
+        } catch (Exception exception) {
+            deleteReport(report);
+            throw new ServerException(exception.getMessage(), exception);
+        }
+
+        return report;
+    }
+
+    private void printHeader(File file) throws ServerException {
+        //TODO: rework to getTotalCount() method from JpaUserDao, when it will be merged to master.
+        printRowToReport("Number of all users: " + adminUserDao.getAll(1, 0).getTotalItemsCount() + "\n", file);
+        final CodenvyLicense license;
+        try {
+            license = licenseManager.load();
+            printRowToReport("Number of users licensed: " + license.getNumberOfUsers() + "\n", file);
+            printRowToReport(
+                    "Date when license expires: " + new SimpleDateFormat("dd MMMM yyyy").format(license.getExpirationDate()) + "\n", file);
+        } catch (LicenseException e) {
+            printRowToReport("Failed to retrieve license!\n", file);
+            LOG.error(e.getMessage(), e);
+        }
+    }
+
+    private void printAllUsersInfo(File file) throws ServerException {
+        Page<UserImpl> currentPage = adminUserDao.getAll(30, 0);
+        do {
+            for (UserImpl user : currentPage.getItems()) {
+                printUserInfo(user, file);
+            }
+            final Optional<PageRef> nextPageRefOpt = currentPage.getNextPageRef();
+            if (nextPageRefOpt.isPresent()) {
+                final PageRef nextPageRef = nextPageRefOpt.get();
+                currentPage = adminUserDao.getAll(nextPageRef.getPageSize(), (int)nextPageRef.getItemsBefore());
+            } else {
+                currentPage = null;
+            }
+        } while (currentPage != null);
+    }
+
+    private void printUserInfo(UserImpl user, File file) throws ServerException {
+        //Print error if failed to retrieve the list of his workspaces
+        List<WorkspaceImpl> workspaces;
+        try {
+            workspaces = workspaceManager.getWorkspaces(user.getId());
+        } catch (ServerException e) {
+            printRowToReport("Failed to receive list of related workspaces for user " + user.getEmail() + "!\n", file);
```

Comment:
Add [ERROR] prefix here and in other places when a such kind of info is printed.